<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RoverMainWindow</class>
 <widget class="QMainWindow" name="RoverMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1106</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoverMainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="MainView">
   <widget class="QFrame" name="cameraSwitchFrame">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>0</y>
      <width>391</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>335</width>
       <height>85</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_22">
        <item>
         <widget class="QRadioButton" name="parametricGimbleCamera">
          <property name="text">
           <string>parametric gimble camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="aerialCamera">
          <property name="text">
           <string>aerial camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="clawCamera">
          <property name="text">
           <string>claw camera</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_23">
        <item>
         <widget class="QRadioButton" name="elbowCamera">
          <property name="text">
           <string>elbow camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="hookCamera">
          <property name="text">
           <string>hook camera</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="GPSConvertFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>1081</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget_2">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>20</y>
       <width>311</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QLabel" name="decimalOoutputTitile">
        <property name="text">
         <string>Decimal Output</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="decimalOutput_3">
        <property name="text">
         <string/>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QPushButton" name="submitButon">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>50</y>
       <width>80</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>598</width>
       <height>112</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_10">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="compCharTitle">
            <property name="text">
             <string>Direction (N,S)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dirInput1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="degreeTitle">
            <property name="text">
             <string>Degrees</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="degreeInput1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="minTitle">
            <property name="text">
             <string>Minutes</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="minInput1"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="secondsTitle">
            <property name="text">
             <string>Seconds</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="secondsInput1"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="dirTitle2">
            <property name="text">
             <string>Direction (E,W)</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="dirInput2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="degTitle2">
            <property name="text">
             <string>Degrees</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="degInput2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_9">
          <item>
           <widget class="QLabel" name="minTitle2">
            <property name="text">
             <string>Minutes</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="minInput2"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_21">
          <item>
           <widget class="QLabel" name="secTitle2">
            <property name="text">
             <string>Seconds</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="secInput2"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QFrame" name="mapFrame">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>691</width>
      <height>461</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QFrame" name="positionFrame">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>381</width>
      <height>101</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>10</y>
       <width>341</width>
       <height>89</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_26">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_24">
          <item>
           <widget class="QLabel" name="currentPosTitle">
            <property name="text">
             <string>Current Position</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentPosDisplay">
            <property name="text">
             <string/>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_25">
          <item>
           <widget class="QLabel" name="targetPosTitle">
            <property name="text">
             <string>Target Position</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="targetPosDisplay">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="lastUpdatePosTitle">
          <property name="text">
           <string>Last Updated:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="margin">
           <number>9</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lastUpdatePosDisplay">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1106</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>submitButon</sender>
   <signal>clicked()</signal>
   <receiver>RoverMainWindow</receiver>
   <slot>submitClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>659</x>
     <y>591</y>
    </hint>
    <hint type="destinationlabel">
     <x>552</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>submitClicked()</slot>
 </slots>
</ui>
